client/blocks/image-editor/style.scss

```diff
@@ -4,6 +4,13 @@
 	height: 100%;
 }
 
+.image-editor .notice {
+	// resorted to !important
+	// because there is
+	// no stronger declaration to overwrite .editor-media-modal .notice
+	margin-top: 0 !important;
```

Comment:
To give you a bit more detail, from computed props, current margin is coming in with a specificity of (ids, classes, elements) ( 0, 2, 0) so we need a specificity of ( 0, 3, 0 ) to beat this.

0<img width="292" alt="screen shot 2017-07-21 at 2 37 38 pm" src="https://user-images.githubusercontent.com/1270189/28483446-2b4fbf88-6e22-11e7-8ab8-d24004bb5e96.png">

Notice should accept a className prop, that we can pass additional classes to. See also 
https://gwwar.com/2015/09/10/why-didnt-that-selector-work/ for a refresher on css specificity